beacon_node/execution_layer/src/lib.rs

```diff
@@ -170,8 +168,8 @@ pub enum BlockProposalContents<T: EthSpec, Payload: AbstractExecPayload<T>> {
         payload: Payload,
         block_value: Uint256,
         kzg_commitments: KzgCommitments<T>,
-        blobs: <Payload::Sidecar as Sidecar<T>>::BlobItems,
-        proofs: KzgProofs<T>,
+        blobs: Option<BlobsList<T>>,
+        proofs: Option<KzgProofs<T>>,
```

Comment:
I'm a bit tempted to couple these two options together into a tuple `Option<(BlobList<T>, KzgProofs<T>)` so that we don't have to deal with unwrapping two options and verifying they're both `None` or `Some` everywhere. It would probably be less prone to future bugs I think?